geps/gep-713.md

```diff
@@ -910,7 +910,115 @@ plugin.
 This is already part of the API pattern, but is being lifted to more prominience
 here.
 
-#### Standard status Condition
+#### Standard status struct
+
+Status: Recommended
+
+Policy objects SHOULD use the upstream `PolicyAncestorStatus` struct in their respective
+Status structs. Please see the included `PolicyAncestorStatus` struct, and its use in
+the `BackendTLSPolicy` object for detailed examples. Included here is a representative
+version.
+
```

Comment:
Not exactly sure where this belongs, but I think some content roughly like this would help show that we want _everyone_ to migrate to this approach, and provide some rationale for why it will be beneficial.
```suggestion
This pattern enables different conditions to be set for different "Ancestors"
of the target resource. This is particularly helpful for policies that may be
implemented by multiple controllers or attached to resources with different
capabilities. This pattern also provides a clear view of what resources a
policy is affecting.

For the best integration with community tooling and consistency across
the broader community, we recommend that all implementations transition 
to Policy status with this kind of nested structure.
```

Reply:
Yeah, love it.